rest/route/admin_route_test.go

```diff
@@ -24,6 +25,161 @@ type AdminRouteSuite struct {
 	postHandler MethodHandler
 }
 
+var testSettings = evergreen.Settings{
```

Comment:
is this duplicated from the data package? should we put this in a common testutil/testdata package?

Reply:
Yep put it in a function